dev/io.openliberty.security.fips.internal_fat/fat/src/io/openliberty/security/fips/fat/tests/fips1403/security/utility/FIPS1403SecurityUtilityTests.java

```diff
@@ -0,0 +1,210 @@
+
+        File defaultEnv = new File(installRoot + "/etc/" + DEFAULT_ENV_FILE);
+        assertTrue(defaultEnv.exists());
+        File fipsProfileFile = new File(installRoot +"/etc/" + FIPS_PROFILE_FILE_NAME);
+        assertTrue(fipsProfileFile.exists());
+        server.startServer();
+        checkLogForFipsEnablementMessage(server, expectedProvider);
+        disableFIPS(FIPS_TARGET.INSTALL, null);
+    }
+
+    @Test
+    public void serverEnvFipsTest() throws Exception {
+        ProgramOutput po = machine.execute(installRoot + SEC_UTILITY_COMMAND, new String[] {SEC_CONF_FIPS_COMMAND,OPT_SERVER+ "=" + SERVER_NAME}, installRoot, env);
+        Log.info(thisClass, "serverEnvFipsTest", "Executed securityUtility configureFIPS command: "+ po.getCommand());
+        Log.info(thisClass, "serverEnvFipsTest", "Result: "+ po.getStdout());
+        Log.info(thisClass, "serverEnvFipsTest", "Error: "+ po.getStderr());
+        assertEquals("securityUtility configureFIPS help did not result in expected return code.",0, po.getReturnCode());
+
+        File defaultEnv = new File(server.getServerRoot() + "/"+ SERVER_ENV_FILE);
+        assertTrue(defaultEnv.exists());
+        File fipsProfileFile = new File(server.getServerRoot() + "/resources/security/" + FIPS_PROFILE_FILE_NAME);
+        assertTrue(fipsProfileFile.exists());
+        server.startServer();
+        checkLogForFipsEnablementMessage(server, expectedProvider);
+        disableFIPS(FIPS_TARGET.SERVER, SERVER_NAME);
+    }
+
+    @After
+    public void teardown() throws Exception {
+        server.stopServer();
+        // clean up all the potentially created files
+        File defaultEnv = new File(installRoot + "/etc/" + DEFAULT_ENV_FILE);
+        File fipsDefaultProfileFile = new File(installRoot +"/etc/" + FIPS_PROFILE_FILE_NAME);
+        File serverEnv = new File(server.getServerRoot() + "/" + SERVER_ENV_FILE);
+        File fipsServerProfileFile = new File(server.getServerRoot() + "/" + FIPS_PROFILE_FILE_NAME);
+
+        // As Default env can contain entries for setting JAVA_HOME or additional Java properties, if we have taken a backup, restore between tests
+        // if no default.env exists, just delete
+        // All other files should be deleted
+        if (defaultEnv.exists()){
+            File defaultBackupEnv = new File(installRoot + "/etc/" + DEFAULT_ENV_BACKUP_FILE);
+            if (defaultBackupEnv.exists()){
+                Files.copy(defaultBackupEnv.toPath(), new File(installRoot+"/etc/"+DEFAULT_ENV_FILE).toPath(), StandardCopyOption.REPLACE_EXISTING);
+            } else {
+                defaultEnv.delete();
+            }
+        }
+        if (fipsDefaultProfileFile.exists()) fipsDefaultProfileFile.delete();
+        if (serverEnv.exists()) serverEnv.delete();
+        if (fipsServerProfileFile.exists()) fipsServerProfileFile.delete();
```

Comment:
there is also a `default.env.backup` / `server.env.backup` / `client.env.backup` file that's created whenever we modify the env files.

while non-functional, i think we should cleanup those files as well. maybe just the `default.env.backup` at the very least since that's in the etc directory.